python/cuml/test/dask/test_comms.py

```diff
@@ -19,7 +19,7 @@
 
 from dask.distributed import wait
 
-from cuml.dask.common.comms import CommsContext, worker_state, default_comms
```

Comment:
I’m thinking we should just invoke the raft tests here.

Reply:
I think invoking RAFT tests here is a great idea, but won't work as the tests are right now, it'll require a bit of thinking and fiddling with imports in the RAFT pytests.

Reply:
What do you think about adding an extra symlink for the raft python tests? Something like `python/cuml/tests/raft`

Reply:
Or perhaps we could add the existing raft test dir to the pytest search path.